src/app/mods/grip-position-save/grip-position-save.ts

```diff
@@ -0,0 +1,182 @@
+// TODO :
+// - Use ipcRendererService instead of wGame.addEventListener('storage', ...
+// Can't make ipcRendererService works : this.ipcRendererService is well instantiate
+// If you log this.ipcRendererService you see in Listener on move-grip
+// But this.ipcRendererService.send('move-grip', target) doesn't ùake anything
+
+import { SettingsService } from "@services/settings.service";
+import { TranslateService } from "@ngx-translate/core";
+
+import { Mod } from "../mod";
+import { IpcRendererService } from "@services/electron/ipcrenderer.service";
+
+export class GripPositionSave extends Mod {
+
+  constructor(
+    wGame: any,
+    settings: SettingsService,
+    translate: TranslateService,
+    public ipcRendererService: IpcRendererService
+  ) {
+    super(wGame, settings, translate);
+    this.ipcRendererService = ipcRendererService;
+
+    this.registerEvent(this.wGame.gui.isConnected);
+
+    // This is a temporary fix because ipcRendererService doesn't work
+    // It works only for accounts in same Lindo window, not in multiple Lindo
+    // E.G :
+    // - You have 2 windows (A, B) of 4 accounts (1, 2, 3 ...)
+    // - You move Party in Window A, Account 1
+    // - Party is moved on every accounts logged in Window A, not in Window B
+    wGame.addEventListener('storage', (e) => {
+      const target = e.key.substr(0, e.key.length - 8);
+      this.moveElement(target);
+    });
+  }
+
+  startMod(): void {
+    return;
+  }
+
+  public registerEvent(skipLogin: boolean = false): void {
+    const onCharacterSelectedSuccess = () => {
+
+      // This doesn't work, the event is never fired
+      this.on(this.ipcRendererService, 'move-grip', (event: Event, target: any) => {
+        // Never displayed in console, this.moveElement never called.
+        console.log('Got event! ' + target);
+        this.moveElement(target);
+      });
+
+      // Timeline
+      this.on(this.wGame.gui.timeline, 'dragEnd', () => { this.timelineDragEnd() });
+
+      if (JSON.parse(localStorage.getItem("timelinePosition"))) {
+        setTimeout(() => {
+          this.moveElement('timeline');
+        }, this.getRandomTime(2, 3));
+      }
+
+      // Party
+      this.on(this.wGame.gui.party, 'dragEnd', () => {
+        this.ipcRendererService.send('move-grip', 'party');
+        this.partyDragEnd();
+      });
+
+      if (JSON.parse(localStorage.getItem("partyPosition"))) {
+        setTimeout(() => {
+          this.moveElement('party');
+        }, this.getRandomTime(2, 3));
+      }
+
+      // Observer to move party when visible, because allways pushed at right.
+      const partyObserver = new MutationObserver(() => {
```

Comment:
Mutation observers are process-heavy, if you only wait for the instanciation you can simply wait `gui.on('connected')`

Reply:
Nop, because (don't tell me why/how) Party, roleplayBuffs and Challenge Indicator are pushed at respectively Right, Top, Left even if I `setStyles({left, top})`
I need to `setStyles` **after** the first display (display: block)

I didn't know Mutation observers were process-heavy, I'll try to see if there is any 'opened' or 'displayed' event, and modify my code in consequences.